ci/jenkins-ci/Jenkinsfile

```diff
@@ -0,0 +1,138 @@
+/**
```

Comment:
I don't think there's anything we need to build for this repo? So there's no need to integrate with the CI?

Reply:
The code builds into a `.hpi` file, which is what Jenkins uses.

Reply:
K. prob should include that in the README

Reply:
On second thought, maybe we shouldn't add it to CI. The build takes a lot of time since it downloads a lot of libraries from maven. Currently, the build for this hasn't even finished yet. This is going to block other jobs.

Reply:
Yeah, I think it makes sense. On the other hand, does freedom-mnl download lots of things? In which case we might want to think about hosting this actually externally since our local bandwidth is tight as well.

Reply:
freedom-mnl doesn't download on every build, so it's ok.

Reply:
Hum? Why not?

Reply:
I think. it only downloads if there's an update on a specific package.